data/ships.txt

```diff
@@ -493,6 +493,7 @@ ship "Blackbird"
 		"LP072a Battery Pack"
 		"D23-QP Shield Generator"
 		"Small Radar Jammer"
+		"Luxury Accommodations"
```

Comment:
Glad to see that the Navy prisoners get a comfy transport back into Republic space during FW Prisoner Parole 1 and 2. :)

Reply:
Haha yes, it had occurred to me on my last playthrough that maybe you should need a Brig for that mission. But then again the Navy officers are portrayed as so honorable that they're prisoners only in the loosest sense of the word. :)